docs/guide/headless-cms.md

```diff
@@ -153,7 +159,7 @@ Apostrophe supports both approaches. The in-context editing features and [widget
 **In any Apostrophe REST API single-document `GET` ("get one") request, include the `?render-areas=1` query parameter.** This will return the page document as JSON. Each area field in the response will have a `_rendered` property with a string of HTML instead of the `items` array of widget objects. This could be useful if we mostly want to insert the rendered area HTML into an existing template system.
 
 ```
-GET https://example.rocks/api/v1/@apostrophecms/page/903y5n76e8j:en:published?apikey=000apikey123&render-areas=1
+GET https://example.rocks/api/v1/@apostrophecms/page/903y5n76e8j:en:published?render-areas=1
 ```
 
 If we wanted to get the full page as HTML rendered using the page templates, we could make a `GET` request to the page URL (basically what browsers do). This is only good if we are not trying to insert it into other templates.
```

Comment:
You can add `?aposRefresh=1` to just get the part of the page that is editable in Apostrophe, which could be useful in this situation. Further customization is possible using a custom query parameter and we need to cover that frequently used technique but that's clearly a scope for a new doc ticket.

Reply:
that's covered in the next paragraph.